### PR TITLE
Enable zero-build updates for test and playground pages using depswriter.py

### PR DESCRIPTION
**Pull request by bcjordan**
Make test and playground pages use a Closure Tools depswriter.py built out dependency graph, allowing for no-build updating
- Change some goog.base usages to use alternate syntax to avoid error in strict mode
- Fix some global variables which caused errors in strict mode
- Add a missing require

### CODE REVIEW

File: blockly-core/blocks/functional_parameters.js [hunk 1]
```diff
@@ -26,6 +26,7 @@
 goog.provide('Blockly.Blocks.functionalParameters');
 
 goog.require('Blockly.Blocks');
+goog.require('Blockly.Blocks.variables');
```
Comment (bcjordan) [inline blockly-core/blocks/functional_parameters.js]: Gets referenced below

File: blockly-core/core/ui/block_space/block_space_editor.js [hunk 2]
```diff
@@ -448,14 +448,14 @@ Blockly.BlockSpaceEditor.prototype.svgResize = function() {
   var svgStyle = window.getComputedStyle(svg);
   var svgBorderWidth = 0;
   if (svgStyle) {
-    svgBorderWidth = parseInt(svgStyle.borderLeftWidth, 10) +
-      parseInt(svgStyle.borderRightWidth, 10);
+    svgBorderWidth = (parseInt(svgStyle.borderLeftWidth, 10) || 0) +
```
Comment (bcjordan) [inline blockly-core/core/ui/block_space/block_space_editor.js]: These were causing errors in the test context when the style was `""`

File: blockly-core/core/ui/block_svg/block_svg.js [hunk 3]
```diff
@@ -937,7 +937,6 @@ Blockly.BlockSvg.prototype.renderDrawTop_ = function(renderInfo, rightEdge,
  */
 Blockly.BlockSvg.prototype.renderDrawRight_ = function(renderInfo, connectionsXY,
     inputRows, iconWidth) {
-  var connectionX, connectionY;
```
Comment (bcjordan) [inline blockly-core/core/ui/block_svg/block_svg.js]: Unused

**Comment by Bjvanminnen** [comment]
lgtm

**Pull request merged**